### DISCUSSION TITLE
Add dates (add / update) onto specific prices

### DISCUSSION THREAD

**Discussion by Max84** [PrestaShop/PrestaShop#40157] [Ideas]
#### Is your feature request related to a problem?

Everywhere in Prestashop database, we have added date and updated date. We can see them on products, orders and many thing.
It's usefull for support, debugging on live site. We can known when an action was performed.
We don't have it onto specific_prices.

#### Describe the solution you'd like

Add date_add and date_upd onto specific_price object.

#### Alternatives you've considered

N/A

#### Additional context

N/A

Thanks.

[a comment by an automated account is not shown]

**Comment by hibatallahAouadni** [comment]
Hi @Max84 

Thanks for the suggestion, the Product Team will take a look.
Please stay tuned !

Thanks!

**Comment by hibatallahAouadni** [comment]
Ping @PrestaShop/product-team ,
@Max84  has a great improvement to share, what do you think !

Thanks !

**Comment by hibatallahAouadni** [comment]
Related to #21520

**Comment by PrestaEdit** [comment]
Hi @PrestaShop/product-team,

It's something on which you want to work, have it onto the next version ?
